polygeist/test/polygeist-opt/sycl/convertToLLVMABI.mlir

```diff
@@ -5,9 +5,24 @@
 // CHECK-NEXT:      [[P2M:%.*]] = "polygeist.pointer2memref"([[A0]]) : (!llvm.ptr<i32, 1>) -> memref<?xi32, 1>
 // CHECK-NEXT:      sycl.call([[P2M]]) {Function = @foo, MangledName = @foo} : (memref<?xi32, 1>) -> ()
 
-gpu.module @device_functions {
+gpu.module @module {
```

Comment:
What in this PR caused the name of the GPU module to change?

Reply:
nothing, just want to be consistent and make the name more generic.